heron/scheduler-core/src/java/com/twitter/heron/scheduler/RuntimeManagerMain.java

```diff
@@ -280,7 +280,7 @@ public static void main(String[] args)
         .put(Key.TOPOLOGY_NAME, topologyName);
 
     // build the final config by expanding all the variables
-    Config config = Config.expand(Config.newBuilder()
+    Config config = Config.toLocalMode(Config.newBuilder()
```

Comment:
Agreed I don't like local mode. Client is also overloaded and could cause confusion. Let's discuss in the similar comment above.